frontend/src/app/components/search-form/search-results/search-results.component.html

```diff
@@ -1,10 +1,13 @@
 <div class="dropdown-menu show" *ngIf="results" [hidden]="!results.hashQuickMatch && !results.addresses.length && !results.nodes.length && !results.channels.length">
-  <ng-template [ngIf]="results.blockHeight">
+  <ng-template [ngIf]="results.blockHeight && !results.blockHeightOutOfRange">
     <div class="card-title" i18n="search.bitcoin-block-height">Bitcoin Block Height</div>
     <button (click)="clickItem(0)" [class.active]="0 === activeIdx" type="button" role="option" class="dropdown-item">
       <ng-container *ngTemplateOutlet="goTo; context: { $implicit: results.searchText }"></ng-container>
     </button>
   </ng-template>
+  <ng-template #noBlockFound [ngIf]="results.blockHeight && results.blockHeightOutOfRange">
```

Comment:
We can remove this part.

We never show "Lightning Channel not found" or similar when posting wrong info ... We just display what it matches.